doc/source/coding-style/pep8.rst

```diff
@@ -1,229 +1,148 @@
-.. _best_practices:
+PEP 8
+=====
 
-PEP 8 Best Practices
-====================
-This topic summarizes key points from `PEP8`_ and how
-they apply to PyAnsys libraries. The goal is for PyAnsys libraries to
-be consistent in style and formatting with the `big three`
-data science libraries: `NumPy`_, `SciPy`_, and `pandas`_.
+This topic summarizes the key points from `PEP 8`_ and how they apply to PyAnsys
+libraries. `PEP 8`_ style guideline was devised by the Python community in order
+to increase the readability of Python code. `PEP 8`_ has been adopted by some of
+the most popular libraries within the Python ecosystem such us: `NumPy`_,
+`SciPy`_, and `pandas`_.
 
+.. _PEP 8: https://www.python.org/dev/peps/pep-0008/
 .. _NumPy: https://numpy.org/
 .. _SciPy: https://www.scipy.org/
 .. _pandas: https://pandas.pydata.org/
-.. _PEP8: https://www.python.org/dev/peps/pep-0008/
 
 
 Imports
-~~~~~~~
+-------
+The following lines describe the code style guidelines that apply for the
+``import`` statement.
+
+Imports Location
+~~~~~~~~~~~~~~~~
 Imports should always be placed at the top of the file, just after any
 module comments and docstrings and before module globals and
 constants.  This reduces the likelihood of an `ImportError`_ that
 might only be discovered during runtime.
 
 .. _ImportError: https://docs.python.org/3/library/exceptions.html#ImportError
 
-Instead of:
+.. tabs::
 
-.. code:: python
+   .. tab:: Avoid
 
-   def compute_logbase8(x):
-       import math
-       return math.log(8, x)
+      .. code-block:: python
 
-Use:
+         def compute_logbase8(x):
+             import math
+             return math.log(8, x)
 
-.. code:: python
+   .. tab:: Use
 
-    import math
+      .. code-block:: python
 
-    def compute_logbase8(x):
-        return math.log(8, x)
+         import math
 
 
+         def compute_logbase8(x):
+             return math.log(8, x)
+
+
+Imports Order
+~~~~~~~~~~~~~
 For better readability, group imports in this order:
 
 #. Standard library imports
 #. Related third-party imports
 #. Local application- or library-specific imports
 
-Instead of:
-
-.. code:: python
-
-   import sys
-   import subprocess
-   from mypackage import mymodule
-   import math
-
-   def compute_logbase8(x):
-      return math.log(8, x)
-
-
-Use:
-
-.. code:: python
-
-   import sys
-   import subprocess
-   import math
-   from mypackage import mymodule
-
-   def compute_logbase8(x):
-       return math.log(8, x)
-
-
-You should place imports in separate lines unless they are
-modules from the same package.
-
-Instead of:
-
-.. code:: python
-
-   import sys, math
-   from my_package import my_module
-   from my_package import my_other_module
-
-   def compute_logbase8(x):
-       return math.log(8, x)
-
-Use:
+All imports should be performed in alphabetically order.
```

Comment:
Should we rephrase to say something like "All imports within each import grouping should be performed in alphabetical order."

Reply:
I agree on this.